packages/bpk-component-modal/src/BpkModalDialog-test.js

```diff
@@ -66,6 +67,7 @@ describe('BpkModalDialog', () => {
           closeEvents={closeEvents}
           dialogRef={jest.fn()}
           isIphone={false}
```

Comment:
ha just noticed that there are other redundant props specified 🙄

Reply:
Same with this, it is needed for the modal dialog